common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -1439,6 +1439,22 @@ def test_join_team(self):
         self.teams_page.click_all_topics()
         self.verify_my_team_count(1)
 
+    def test_join_team_from_teams_page(self):
```

Comment:
Note: the existing test for joining teams doesn't fail because it visits the page directly. The bug is only triggered when navigating to the teams page through the UI.